### PR TITLE
[secureSign] add createTree image, OPENSHIFT env variable

### PR DESCRIPTION
**Pull request by fghanmi**
add createTree image &  OPENSHIFT env variable: used by e2e tests running on OCP environments.

### CODE REVIEW

**Comment by fghanmi** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

File: ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml [hunk 1]
```diff
@@ -144,6 +145,7 @@ tests:
         export TEST_IMAGE=ttl.sh/$PROW_JOB_ID:1h
         go install github.com/sigstore/cosign/v2/cmd/cosign@v2.2.4
         make test-e2e
+        export OPENSHIFT=true
```
Comment (osmman) [inline ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml:148]: you have to put it before execution of e2e tests

**Comment by JasonPowr** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

File: ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml [hunk 2]
```diff
@@ -76,6 +76,7 @@ tests:
         {"name": "CLIENT_SERVER_F_IMAGE", "value": $(cat internal/controller/constants/images.go | grep "ClientServerImage_f" | grep -o '[^[:space:]]*@sha.*' || echo "null")},
         {"name": "SEGMENT_BACKUP_JOB_IMAGE", "value": $(cat internal/controller/constants/images.go | grep "SegmentBackupImage" | grep -o '[^[:space:]]*@sha.*')},
         {"name": "TIMESTAMP_AUTHORITY_IMAGE", "value": $(cat internal/controller/constants/images.go | grep "TimestampAuthorityImage" | grep -o '[^[:space:]]*@sha.*' || echo "null")}
```
Comment (JasonPowr) [inline ci-operator/config/securesign/secure-sign-operator/securesign-secure-sign-operator-main.yaml:78]:
```suggestion
        {"name": "TIMESTAMP_AUTHORITY_IMAGE", "value": $(cat internal/controller/constants/images.go | grep "TimestampAuthorityImage" | grep -o '[^[:space:]]*@sha.*' || echo "null")},
```

**Comment by JasonPowr** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Comment by JasonPowr** [comment]
/pj-rehearse pull-ci-securesign-secure-sign-operator-main-tas-operator-e2e

**Review by JasonPowr — APPROVED** [review]

**Pull request merged**